res/controllers/Keyboard-Default.kbd.xml

```diff
@@ -0,0 +1,388 @@
+      <control action="rate_temp_up_small">
+        <keyseq>Shift+F4</keyseq>
+      </control>
+      <control action="reloop_exit">
+        <keyseq byPositionOf="en_US">4</keyseq>
+      </control>
+      <control action="reverse">
+        <keyseq byPositionOf="en_US">Shift+A</keyseq>
+      </control>
+      <control action="vinylcontrol_cueing">
+        <keyseq byPositionOf="en_US">Ctrl+Alt+Y</keyseq>
+      </control>
+      <control action="vinylcontrol_mode">
+        <keyseq byPositionOf="en_US">Ctrl+Shift+Y</keyseq>
+      </control>
+    </group>
+    <group name="[Channel2]">
+      <control action="LoadSelectedTrack">
+        <keyseq>Shift+Right</keyseq>
+      </control>
+      <control action="back">
+        <keyseq byPositionOf="en_US">j</keyseq>
+      </control>
+      <control action="beatloop_4_toggle">
+        <keyseq byPositionOf="en_US">u</keyseq>
+      </control>
+      <control action="beatsync">
+        <keyseq byPositionOf="en_US">6</keyseq>
+      </control>
+      <control action="bpm_tap">
+        <keyseq byPositionOf="en_US">Shift+^</keyseq>
+        <keyseq lang="ru_RU">Shift+,</keyseq>
+        <keyseq lang="es_ES">Shift+/</keyseq>
+      </control>
+      <control action="cue_default">
+        <keyseq byPositionOf="en_US">;</keyseq>
+
+        <!-- Scancode: TODO: No scancode found in el_GR for character: '&#180;'-->
+        <keyseq lang="el_GR">´</keyseq>
+      </control>
+      <control action="cue_gotoandstop">
+        <keyseq byPositionOf="en_US">Shift+:</keyseq>
+        <keyseq lang="el_GR">Shift+&quot;</keyseq>
+      </control>
+      <control action="cue_set">
+        <keyseq byPositionOf="en_US">Shift+L</keyseq>
+        <keyseq lang="da_DK">Shift+!</keyseq>
+      </control>
+      <control action="eject">
+        <keyseq>Alt+Shift+Right</keyseq>
+      </control>
+      <control action="filterLowKill">
+        <keyseq byPositionOf="en_US">n</keyseq>
+      </control>
+      <control action="flanger">
+        <keyseq byPositionOf="en_US">0</keyseq>
+
+        <!-- Scancode: TODO: No scancode found in cs_CZ for character: '0'-->
+        <keyseq lang="cs_CZ">0</keyseq>
+      </control>
+      <control action="fwd">
+        <keyseq byPositionOf="en_US">k</keyseq>
+      </control>
+      <control action="hotcue_1_activate">
+        <keyseq byPositionOf="en_US">m</keyseq>
+      </control>
+      <control action="hotcue_1_clear">
+        <keyseq byPositionOf="en_US">Shift+M</keyseq>
+      </control>
+      <control action="hotcue_2_activate">
+        <keyseq byPositionOf="en_US">,</keyseq>
+      </control>
+      <control action="hotcue_2_clear">
+        <keyseq byPositionOf="en_US">Shift+&lt;</keyseq>
+        <keyseq lang="es_ES">Shift+¿</keyseq>
+      </control>
+      <control action="hotcue_3_activate">
+        <keyseq byPositionOf="en_US">.</keyseq>
+      </control>
+      <control action="hotcue_3_clear">
+        <keyseq byPositionOf="de_DE">Shift+:</keyseq>
```

Comment:
`en_US` is one of the two layouts that have two keys sharing the same keysym for the same modifier Check [here](https://github.com/mixxxdj/mixxx/pull/966#discussion_r75596934) for explanation.

This means that `en_US` users will have this action mapped to both keys, which I don't think it's a problem because the `<LSGT>` key is not autochthonal to American keyboards. It's the key between the left Shift and the Z key, which does only exist on European keyboards, a.k.a the 105th key.